### PR TITLE
optimize with str.format(**dict)

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: soccer/writers.py [hunk 1]
```diff
@@ -135,49 +129,20 @@ def standings(self, league_table, league):
...
-            el_upper = LEAGUE_PROPERTIES[league]['el'][0]
-            el_lower = LEAGUE_PROPERTIES[league]['el'][1]
-            rl_upper = LEAGUE_PROPERTIES[league]['rl'][0]
-            rl_lower = LEAGUE_PROPERTIES[league]['rl'][1]
+            cl_upper, cl_lower = LEAGUE_PROPERTIES[league]['cl']
+            el_upper, el_lower = LEAGUE_PROPERTIES[league]['el']
+            rl_upper, rl_lower = LEAGUE_PROPERTIES[league]['rl']
 
+            team_str = ("{position:7} {teamName:33} {playedGames:12}"
+               " {goalDifference:14} {points}").format(**team)
```
Comment (carlosvargas) [inline soccer/writers.py]: The padding is a little off for `position` and `played`
image

**Comment by carlosvargas** [comment]
I got an unicode error while trying to run this:

    > python -m soccer.main --players --team AFC
     File "soccer\writers.py", line 118, in team_players
        click.secho(fmt.format(**player), bold=True)
     UnicodeEncodeError: 'ascii' codec can't encode character u'\u20ac' in position 11: ordinal not in range(128)

Therefore, we'll need to make sure the strings are unicode:

        fmt = (u"{jerseyNumber:4} {name:28} {position:23} {nationality:23}"
               u" {dateOfBirth:18} {marketValue}")

Using `__future__ import unicode_literals` seems to be throwing an error somewhere else (I don't have the time right now to debug it), but if you can fix that, I think using this might be better.

**Comment by cclauss** [comment]
I made the suggested changes.  Sorry about that.

**Comment by carlosvargas** [comment]
Thanks. If you fix the padding issues, then we'll be good.

Before:
image

Now: 
image

**Comment by cclauss** [comment]
str.format() left aligns text but right aligns numbers.  In this last mod, I forced everything to be left aligned.

**Comment by carlosvargas** [comment]
Thanks @cclauss! :+1:

**Pull request merged**